cli/blockchain_info/help.py

```diff
@@ -0,0 +1,12 @@
+"""
+Provide help messages for command line interface's blockchain info commands.
+"""
+GET_BATCH_ID_ARGUMENT_HELP_MESSAGE = 'Get batch by id.'
+
+GET_BATCH_START_ARGUMENT_HELP_MESSAGE = 'Get batch by start.'
+
+GET_BATCH_LIMIT_ARGUMENT_HELP_MESSAGE = 'Limit amount of batches.'
+
+GET_BATCH_HEAD_ARGUMENT_HELP_MESSAGE = 'Get batch by head.'
+
+GET_BATCH_REVERSE_ARGUMENT_HELP_MESSAGE = 'Reverse result.'
```

Comment:
Remove blank lines between the same entities.

Reply:
Done.